cmd/lakefs-loadtest/cmd/db.go

```diff
@@ -23,15 +17,6 @@ var dbCmd = &cobra.Command{
 	Short: "Load test database actions",
 }
 
-func connectToDB(ctx context.Context, connectionString string) db.Database {
-	database, err := db.ConnectDB(ctx, params.Database{Driver: db.DatabaseDriver, ConnectionString: connectionString})
-	if err != nil {
-		fmt.Printf("Failed connecting to database: %s\n", err)
-		os.Exit(1)
-	}
-	return database
-}
-
 //nolint:gochecknoinits,gomnd
 func init() {
 	rootCmd.AddCommand(dbCmd)
```

Comment:
We don't need the db flag anymore right?

Reply:
Also, I don't really understand - it seems like this command is doing nothing?

Reply:
It's the baseline for `entry` subcommand of `loadtest`. 
I think we need the flag if we want to keep this command.